spec/features/purchases_show_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'when opening the purchase details page', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Tom', email: 'tom@example.com', password: 'topsecret')
+    @user.confirm
+
+    visit new_user_session_path
+    fill_in 'Email', with: 'tom@example.com'
+    fill_in 'Password', with: 'topsecret'
+    click_button 'Log in'
+
+    @group1 = Group.create(user: @user, name: 'Food', icon: 'https://i.pravatar.cc/300?img=13')
+    @group2 = Group.create(user: @user, name: 'Cosmetics', icon: 'https://i.pravatar.cc/300?img=1')
+    @purchase1 = Purchase.create(name: 'Apples', amount: 6, author: @user, groups: [@group1, @group2])
+    @purchase2 = Purchase.create(name: 'Bananas', amount: 4, author: @user, groups: [@group1])
+    visit(group_purchase_path(@group1, @purchase1))
+  end
+
+  context 'shows the right content' do
+    it 'heading' do
+      expect(page).to have_content('details')
```

Comment:
- [ ] As an example, this spec is failing, due to a case sensitivity issue. There is "Details" on the page but you are checking for "details" so it fails.

  ![image](https://github.com/katarighe/budget-app/assets/81468675/ae9ffe80-a1a4-41b6-972e-5f3f8b4b9571)

Reply:
✅ Solved